framework/CustomModes/MPISimulationMode.py

```diff
@@ -55,6 +55,13 @@ def modifyInfo(self, runInfoDict):
       @ In, runInfoDict, dict, the original runInfo
       @ Out, newRunInfo, dict, of modified values
     """
+    magicSyntax = False
```

Comment:
I think these modifications are in the RAY branch.... I would toss them out

Reply:
Thanks Andrea, I have resolved the conflict of ray branch, but there are several issues happened to the PR. We will try to resolve and merge the ray pr, and then we will rebase this branch.

Reply:
The MPISimulationMode changes should be removed.

Reply:
removed